nasa-veda-platform/dashboard/index.qmd

```diff
@@ -1,75 +1,39 @@
+VEDA Dashboards can serve as the main representation of scientific programs and organizations, putting the data and results in the context of the program, while using open community standards for data access, such that results become directly reproducible and the data access interoperable with other tools such as GIS software or computing environments.
 
-### 3. It enables science workflows AND communicates to stakeholders in one platform
-- We show that, with cloud-optimized data, we can achieve multiple purposes without considerable additional effort or duplicating data sources
-- Data visualization in web interfaces often relies on copies of original data that were prepared for visualization (even scalable visualization services like NASA WorldView rely on statically produced rendered replicas). Linking the visualization back to original source data, as required for (re)producing scientific analysis, is often not possible or requires connecting a lot of dots.
-- VEDA visualizes data directly off a single authoritative reference that contains data optimized for visualization as well as direct access, avoiding the issue of missing links between source and rendered data. The data a user sees in VEDA has a direct reference to source data (often via STAC).
-Another benefit of dynamic tiling is that users are able to adjust viewing parameters such as color map and value range to focus on, to study details of their interest.
+The VEDA Dashboard powers several platforms, most prominently the [U.S. Greenhouse Gas Center](https://earth.gov/ghgcenter) and [Earth.gov](https://earth.gov), with more platform instances underway.
 
-### 4. Features are optimized for Earth science data but other spatio-temporal applications might find them useful
-- Spatio-temporal data often comes from Earth observations or Earth system models. However, the principle of integrating dynamically rendered data into a website is valuable for many branches of data-intensive science, such as heliophysics or astrophysics
+See also this detailed list of [principles the VEDA Dashboard is designed around](./design-principles.qmd).
 
-### 5. The VEDA Dashboard is completely open-source
-- The code is open and accessible, and the tool itself proliferates the use of open standards and reproducible data access for open science
-Community involvement will become important to the long-term success of VEDA
 
-### 6. Each science team can create their own VEDA Dashboard with different content, data, and features
-Existing VEDA instances:
+## Made for reuse
 
-- Template
-- Earthdata
-- Greenhouse Gas Center Portal
-- earth.gov
-
-Planned VEDA instances:
+If you would like to learn more about how to create your own instance of the VEDA Dashboard, visit the [VEDA UI docs](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/development/SETUP.md).
```

Comment:
Very soon I will want to send them to another page in veda-docs, working on it with this PR: https://github.com/NASA-IMPACT/veda-docs/pull/182